### PR TITLE
added and updated initial group samples

### PR DESCRIPTION
**Pull request by sanana30**

### CODE REVIEW

File: sample-queries/sample-queries.json [hunk 1]
```diff
@@ -204,7 +204,69 @@
...
+    },
+    {
+        "id": "b15626bb-d583-4505-a7ee-85f48f1710f4",
+        "category": "Groups",
+        "method": "POST",
+        "humanName": "create a new group",
+        "requestUrl": "/v1.0/groups",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/group-post-groups?view=graph-rest-1.0&tabs=http",
+        "tip": "Please provide us with feedback on the groups API here: https://aka.ms/GroupsAPIFeedback",
+        "postBody": "{\r\n  \"displayName\": \"Library Assist\",\r\n  \"mailEnabled\": true\",\r\n  \"mailNickname\": \"library\",\r\n  \"securityEnabled\": true\"\r\n}",
```
Comment (irvinesunday) [inline sample-queries/sample-queries.json:236]:
```suggestion
        "postBody": "{\r\n  \"description\": \"Self help community for library\",\r\n \"displayName\": \"Library Assist\",\r\n  \"groupTypes\": [ \r\n \"Unified\" \r\n],\r\n \"mailEnabled\": true\",\r\n  \"mailNickname\": \"library\",\r\n  \"securityEnabled\": true\"\r\n}",
```

**Review by irvinesunday — APPROVED** [review]
👍🏽

File: sample-queries/sample-queries.json [hunk 2]
```diff
@@ -204,7 +204,69 @@
...
+        "skipTest": false
+    },    
+    {
+        "id": "c165c350-26ab-41bb-97d6-0e8b3fec051e",
+        "category": "Groups",
+        "method": "DELETE",
+        "humanName": "remove member from group",
+        "requestUrl": "/v1.0/groups/{id}/members/{id}/$ref",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/group-delete-members?view=graph-rest-1.0&tabs=http",
+        "tip": "Please provide us with feedback on the groups API here: https://aka.ms/GroupsAPIFeedback",
```
Comment (bettirosengugi) [inline sample-queries/sample-queries.json:257]: Might be great to also add an additional tip message letting users know how to get the {id} described.

Also the {id should be descriptive e.g. {message-id}

Please do the same for all queries that need an {id}

**Review by bettirosengugi — CHANGES_REQUESTED** [review]

File: sample-queries/sample-queries.json (hunk 2 above)
Comment (sanana30) [reply sample-queries/sample-queries.json:257]: Thanks @bettirosengugi - I've updated the {id} to make it more descriptive,

Comment (bettirosengugi) [reply sample-queries/sample-queries.json:257]: This is great @sanana30 please add an extra line on the tip message letting users know which request url they need to run to get the id e.g "To get group-id run http://graph.microsoft.com//v1.0/groups/ "

**Review by bettirosengugi — CHANGES_REQUESTED** [review]

File: sample-queries/sample-queries.json (hunk 2 above)
Comment (sanana30) [reply sample-queries/sample-queries.json:257]: @bettirosengugi - Would this need to be added to the same tip message that has our survey link? Or would I add a new tip message?

Comment (bettirosengugi) [reply sample-queries/sample-queries.json:257]: @sanana30 same tip message. Just add a period at the end of the feedback string then write the other.

**Comment by sanana30** [comment]
@bettirosengugi thanks for the suggestion. I've made the changes now

**Review by bettirosengugi — APPROVED** [review]

**Pull request merged**